quickget

```diff
@@ -1503,6 +1513,25 @@ function get_manjaro() {
     echo "${URL} ${HASH}"
 }
 
+function get_miyo() {
+    local EDITION="${1:-}"
+    local HASH=""
+    local DATE=""
+    local ISO="miyo-${EDITION}-x86_64-BIOS-${DATE}.iso"
```

Comment:
AS DATE is unset this will be broken and is also overwritten in the `case` below so please remove the ISO assignment and the unused DATE (or assign and use it in the EDITION `case` options )